Realm/RLMSchema.mm

```diff
@@ -126,12 +126,20 @@ - (instancetype)init {
 
 - (NSArray *)objectSchema {
     if (!_objectSchema) {
+        if (self == s_sharedSchema) {
```

Comment:
This check is only safe if `s_sharedSchema` is never assigned to after startup, so the variable should be made `const` (which requires putting the `const` to the right of the `*` in the declaration).